### PR TITLE
PTRef: trips api

### PR DESCRIPTION
**Pull request by pbougue**
JIRA: http://jira.canaltp.fr/browse/NAVP-429

### CODE REVIEW

File: source/disruption/traffic_reports_api.cpp [hunk 1]
```diff
@@ -187,7 +187,7 @@ void TrafficReport::add_vehicle_journeys(const std::vector<type::idx_t>& network
         } catch (const ptref::parsing_error& parse_error) {
             LOG4CPLUS_WARN(logger, "Disruption::add_stop_areas : Unable to parse filter "
                            << parse_error.more);
-        } catch (const ptref::ptref_error& ptref_error) {
+        } catch (const ptref::ptref_error& /*ptref_error*/) {
```
Comment (TeXitoi) [inline source/disruption/traffic_reports_api.cpp]: I think you can just remove the name of the variable.

**Comment by TeXitoi** [comment]
OK for me.

File: source/ptreferential/tests/ptref_test.cpp [hunk 2]
```diff
@@ -532,7 +532,7 @@ BOOST_AUTO_TEST_CASE(mvj_filtering) {
 
     // looking for MetaVJ 0
     indexes = make_query(nt::Type_e::MetaVehicleJourney,
-                         "meta_vehicle_journey.uri=\"vehicle_journey 0\"",
+                         "trip.uri=\"vehicle_journey 0\"",
```
Comment (xlqian) [inline source/ptreferential/tests/ptref_test.cpp]: `R"(trip.uri="vehicle_journey 0")"`

Comment (pbougue) [inline source/ptreferential/tests/ptref_test.cpp]: Good to know ;)

**Comment by xlqian** [comment]
Ok for me

**Comment by pbougue** [comment]
Jenkins, test this please

**Comment by pbougue** [comment]
Jenkins, test this please

File: source/type/type.cpp [hunk 3]
```diff
@@ -358,7 +358,6 @@ static bool concerns_base_at_period(const VehicleJourney& vj,
             if (*titr < concerned_vp.beginning_date) { continue; }
             size_t day((*titr - concerned_vp.beginning_date).days());
             if (day >= concerned_vp.days.size() || ! concerned_vp.check(day)) { continue; }
-            if (*titr < concerned_vp.beginning_date) { continue; }
```
Comment (pbougue) [inline source/type/type.cpp:361]: :wink:

**Pull request merged**